TU-Agent/test.test2g

```diff
@@ -6,11 +6,39 @@ use tygron as actionspec.
 
 timeout = 30.
 
-%We're running into very strange problems with this file in travis
 test tygron with
 	pre {goal(indicatorGoal(_, _))}
 	in {
-		%apparently travis doesn't allow us to test the in clauses
+		%Percept tests%
+		%We only percept this only once because of the empty list believe.
+		percept(buildings(X)), bel(buildings([])) leadsto bel(buildings(X)).
+		%Tests for percepts that are only send on change, according to template P-on-change
+		percept(stakeholders(X)) leadsto bel(stakeholders(X)).
+		percept(stakeholders(X)), bel(stakeholders(Y), not(X=Y)) leadsto not(bel(stakeholders(Y))).
```

Comment:
Maybe leadsto, to a new line for consistency with the other tests and for readability?